locales/nl_NL/pcsx2_Iconized.po

```diff
@@ -0,0 +1,871 @@
+# Translations for PCSX2 (Iconized).
+# Copyright (C) 2018 PCSX2 Dev Team
+# This file is distributed under the same license as the PCSX2 package.
+# First author: mourix, 2018
```

Comment:
Add an extra line with # after this one to both files.
```

Should be 
# Translations for PCSX2 (Iconized).
# Copyright (C) 2018 PCSX2 Dev Team
# This file is distributed under the same license as the PCSX2 package.
# First author: mourix, 2018
#
```

Reply:
Done.